geowebcache/rest/src/main/java/org/geowebcache/rest/config/GWCRestConfig.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author David Vick / Boundless 2017
+ */
+package org.geowebcache.rest.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurationSupport;
+
+/**
+ * Configure various aspects of Spring MVC
+ */
+@Configuration
+@EnableWebMvc
+public class GWCRestConfig extends WebMvcConfigurationSupport {
```

Comment:
So, this class doesn't do anything right now... Any particular reason why it is here?

Reply:
No, this class can be removed at this point.

Reply:
So, after talking with @dvntucker , looks like the `@EnableWebMvc` here is doing the same thing that `<mvc:annotation-driven />` was, so when we remove this we will need to re-add that to the `geowebcache-rest-context.xml` (GeoServer has a Config class with the `@EnableWebMvc` annotation)
Since this is in WEB-INF, it may make GWC play nicer with the GS config.